### PR TITLE
Add man page for 'bundle issue' command

### PR DESCRIPTION
**Pull request by andrew**
## What was the end-user or developer problem that led to this PR?

Missing man page for `bundle issue` command https://github.com/rubygems/rubygems/issues/8268

## What is your fix for the problem, implemented in this PR?

Add a man page for `bundle issue` command 

## Make sure the following tasks are checked

- [x] Describe the problem / feature
- [ ] Write [tests](https://github.com/rubygems/rubygems/blob/master/bundler/doc/development/PULL_REQUESTS.md#tests) for features and bug fixes
- [x] Write code to solve the problem
- [x] Make sure you follow the [current code style](https://github.com/rubygems/rubygems/blob/master/doc/bundler/development/PULL_REQUESTS.md#code-formatting) and [write meaningful commit messages without tags](https://github.com/rubygems/rubygems/blob/master/doc/bundler/development/PULL_REQUESTS.md#commit-messages)

### CODE REVIEW

File: bundler/lib/bundler/man/bundle-issue.1.ronn [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+The command outputs several sections:
+
+* Troubleshooting links and resources
+* Link to the GitHub issue template
+* Environment information including:
+    * Bundler version and platforms
+    * Ruby version and configuration
+    * RubyGems version and paths
+    * Development tool versions (Git, RVM, rbenv, chruby)
```
Comment (deivid-rodriguez) [inline bundler/lib/bundler/man/bundle-issue.1.ronn:27]: It seems nronn does not generate correct man output for double nested list. This is the ronn-ng [ticket](https://github.com/apjanke/ronn-ng/issues/16) but it's the same for nronn. I guess include the nested list information directly as an enumeration:

> * Environment information including Bundler version and platforms, Ruby version and configuration, RubyGems version and paths, and Development tool versions (Git, RVM, rbenv, chruby)

File: bundler/lib/bundler/man/index.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#                     bundle-issue.1
```
Comment (deivid-rodriguez) [inline bundler/lib/bundler/man/index.txt:1]: You run into the same issue again, I guess regenerate it like you did before.

**Review by deivid-rodriguez — COMMENTED** [review]
Thank you!

**Comment by andrew** [comment]
Running `bin/rake man:build` again keeps reverting back to:

```
#                     bundle-issue.1
```

Weird!

**Comment by deivid-rodriguez** [comment]
Indeed now `bin/rake man:build` in CI seems to be having the same issue you're having locally?

File: bundler/lib/bundler/man/bundle-issue.1.ronn [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# bundle-issue(1) -- Get help reporting Bundler issues
```
Comment (deivid-rodriguez) [inline bundler/lib/bundler/man/bundle-issue.1.ronn:1]: This was the problem with the `index.txt` generation:

```suggestion
bundle-issue(1) -- Get help reporting Bundler issues
```

**Review by deivid-rodriguez — COMMENTED** [review]
Found the issue!

**Comment by andrew** [comment]
cleaned up the whitespace

**Pull request merged**